services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss

```diff
@@ -32,3 +31,8 @@
     width: 50%;
   }
 }
+
+.resources-heading {
+  margin-top: spacing.$spacing-11;
+  margin-bottom: spacing.$spacing-07;
+}
```

Comment:
```suggestion
```
I don't think we need these styles

Reply:
woops! my bad I thought I commited them the first time 🤦‍♀️